test/examples/cell_expansion.qmd

```diff
@@ -0,0 +1,49 @@
+---
+title: Cell Expansion
+---
+
+```{julia}
+#| echo: false
+#| multiple: true
+[(;
+    thunk = function ()
+        println("print call")
+        display("display call")
+        "return value"
+    end,
+    code = """
+    # Fake code goes here.
```

Comment:
maybe we need to also guard against people adding more `#|` front matter commands here?

Reply:
Perhaps. What are we guarding them against, though?

Reply:
weird interactions with the options?